remme/tests/test_rest_api.py

```diff
@@ -0,0 +1,97 @@
+import time
```

Comment:
Those dumb transactions will definitely fail. Why don't we test the API against a particular correct transaction and test if it works?

Reply:
I will add func tests.